### PR TITLE
MC-1460: add File.trimBox so that suppliers can easily see the flat dimensions of folded items

### PR DESCRIPTION
**Pull request by drewwills**
https://mixam.atlassian.net/browse/MC-1460

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -146,11 +148,22 @@ The top level of the MxJdf document contains the following elements:
...
+Specifies the dimensions of the component in localised units (i.e. mm or inches). These are
+ordinarily the dimensions at which the article was sold; so in the case of an A5 hardcover book,
+both the body and the cover components would have 210 for the `longEdge` and 148 for the
+`shortEdge`. Also in the case of folded products, if the article was sold to the customer using flat
+sizes (US & Canada) these dimensions will match the flat size of the article; in other locales these
+dimensions will match the folded size.
+
+If the article contains a dust jacket component, by contrast, these dimensions on that component
+will match the trim box of the actual artwork for the just jacket.
+
```
Comment (acolebourne) [inline README.md:161]: Do we know if this is in use? It seems bit hard to follow!

File: README.md [hunk 2]
```diff
@@ -467,7 +482,23 @@ See type ‘Address’
...
+    "longEdge": 210,
+    "shortEdge": 99,
+    "units": 0
+}
+```
+
+| Property | Description | Values |
+|---|---|---|
+|longEdge|Component long axis. (Double precision number) in shop’s units||
+|shortEdge|Component short axis. (Double precision number) in shop’s units ||
```
[the remark this answers is not in the document]
Comment (drewwills) [reply README.md:497]: I'm not in love with saying it this way, but I do view it as a cross-platform standard: https://en.wikipedia.org/wiki/Double-precision_floating-point_format
